### PR TITLE
completed introduction to git and group assignment

### PR DESCRIPTION
**Pull request by kpeters**
after a group assignment has been completed you can do a hard reset followed by push --force

### CODE REVIEW

File: Group_assignment/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Group assignment
+To complete the following assignment you need to work in groups, preferrably pairs  
+In the following sections we'll note if an action is to be performed individually or as a group.
+
+## First get a local copy (individually)
+Begin by clicking the **Fork** button in the header of the repository    
+After Forking is complete, click the **Clone** button, copy the url for either HTTPS or SSH  
+In GitBash, navigate to the directory where you keep al your git repos by entering the following command:
```
Comment (mark-de-haan) [inline Group_assignment/README.md]: keep *all of your* git repos

File: Group_assignment/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+```cd ~\Documents\git```  
+Once in the git directory, clone the repo by entering ```git clone <paste url here>```, you now have a local copy
+
+## The assignment
+### Start making changes
+Each group should be editing their own file, align beforehand which group will edit which file.
```
Comment (mark-de-haan) [inline Group_assignment/README.md:14]: which person is going to edit which file?

File: Group_assignment/README.md (hunk 1 above)
Comment (kpeters) [reply Group_assignment/README.md]: Good catch, will fix

File: Group_assignment/README.md (hunk 2 above)
Comment (kpeters) [reply Group_assignment/README.md:14]: The members of group 1 are going to edit group1.py, mebers of group 2 edit group2.py etc. I think this line is fine.

Comment (mark-de-haan) [reply Group_assignment/README.md:14]: Ok got it

**Pull request merged**